### PR TITLE
[Lang] Support very basic python string.format() now

### PR DESCRIPTION
**Pull request by Leonz5288**
Related issue = #2396

### CODE REVIEW

**Comment by Leonz5288** [comment]
We support basic string format in print now. For example, code below:
`import taichi as ti
ti.init()
ops = ti.field(ti.float32, shape=(1, ))
yes = ti.field(ti.int32, shape=(1, ))

@ti.kernel
def test():
    ops[0] = 3.14
    yes[0] = 3
    print('Two numbers:', ' {} and {} '.format(ops[0], yes[0]), 'and a list: ', ['q', 'w', 'e', 'r'], sep='|')
    print('round{}square{}what?{}'.format(ops[0], 123, ' wellwellwell'))

test()`
Would print the result:
`Two numbers:| 3.140000 and 3 |and a list: |[q, w, e, r]
round3.140000square123what? wellwellwell`

File: python/taichi/lang/transformer.py [hunk 1]
```diff
@@ -626,6 +626,13 @@ def visit_Call(self, node):
         if not ASTResolver.resolve_to(node.func, ti.static, globals()):
             # Do not apply the generic visitor if the function called is ti.static
             self.generic_visit(node)
+        if isinstance(node.func, ast.Attribute):
+            attr_name = node.func.attr
+            if attr_name == 'format':
+                node.args.insert(0, node.func.value)
+                node.func = self.parse_expr('ti.ti_format')
+            else:
+                pass
```
Comment (ljcc0930) [inline python/taichi/lang/transformer.py:635]:
```suggestion
```

**Review by ljcc0930 — APPROVED** [review]
LGTM!

**Pull request merged**